.github/workflows/dependabot-automerge.yml

```diff
@@ -1,28 +1,28 @@
-# **** AUTOMERGE ****
-# Merge automatically the PR that contain a minor or patch update on the dependency you define in env.DEPENDENCY
-#  - Inspiration: https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/automating-dependabot-with-github-actions#enable-auto-merge-on-a-pull-request
-
-name: Dependabot auto-merge
+name: Dependabot auto-approve
 on: pull_request
 
 permissions:
   pull-requests: write
-  contents: write
 
 jobs:
   dependabot:
     runs-on: ubuntu-latest
-    if: ${{ github.actor == 'dependabot[bot]' }}
+    if: github.event.pull_request.user.login == 'dependabot[bot]' && github.repository == 'storyblok/storyblok-react'
     steps:
       - name: Dependabot metadata
         id: metadata
-        uses: dependabot/fetch-metadata@v1.1.1
+        uses: dependabot/fetch-metadata@v2
         with:
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
-      - name: Enable auto-merge for Dependabot PRs
-        if: ${{contains(steps.metadata.outputs.dependency-names, env.DEPENDENCY) && (steps.metadata.outputs.update-type == 'version-update:semver-patch' || steps.metadata.outputs.update-type == 'version-update:semver-minor')}}
-        run: gh pr merge --auto --merge "$PR_URL"
+          github-token: '${{ secrets.GITHUB_TOKEN }}'
+          alert-lookup: true
+      - uses: actions/checkout@v4
+      - name: Approve a PR if not already approved
+        run: |
+          gh pr checkout "$PR_URL" # sets the upstream metadata for `gh pr status`
+          if [ "$(gh pr status --json reviewDecision -q .currentBranch.reviewDecision)" != "APPROVED" ];
+          then gh pr review --approve "$PR_URL"
+          else echo "PR already approved, skipping additional approvals to minimize emails/notification noise.";
+          fi
         env:
-          DEPENDENCY: "@storyblok/js"
```

Comment:
@edodusi what's the reason to remove the `DEPENDENCY`? wouldn't it be cleaner than hardcode the repo/package above as a string?

Reply:
the above steps come straight from the GitHub official documentation so I thought to include them as best practice

Reply:
I see. I still see the env cleaner. At the end, that's the only part we need to change from one SDK to another and it's easier to identify. But it's more of a style thing, not a blocker